aine/templates/wp-custom-template-landing-page.html

```diff
@@ -0,0 +1,15 @@
+<!-- wp:template-part {"slug":"header-landing-page"} /-->
```

Comment:
I know it sounds like a broken record 😅 , but I'm not sure if we should ship a custom template to achieve a unique home page design.

Reply:
Yes, I was not 100% sure about what to do here. Maybe I could build this as an index, leaving the posts and include the description at the bottom in the footer, but I wonder if that's even more confusing.

Reply:
As conversations progress, `front-page.html` that contains a post content block (like you've done with this custom template) could end up being the way to go here, when themes want to supply a unique template for their home page design.

Reply:
Maybe we should use index.html here so we can progress this theme?

Reply:
Agreed, I moved it to index for now and got rid of the custom homepage template. @beafialho does this work for you?